addons/gamification/models/gamification_challenge.py

```diff
@@ -645,6 +651,31 @@ def discard_challenge(self):
         sudoed.message_post(body=_("%s has refused the challenge", user.name))
         return sudoed.write({'invited_user_ids': (3, user.id)})
 
+    @api.model
+    @tools.ormcache()
+    def _get_cron_update_interval_or_default(self):
+        """Return minimum interval in seconds between cron updates of challenges goals.
+
+        Returns 86_400 if cron record was not found (= 1 day, default interval for this cron).
+        """
+        cron_interval_data = (
+            self.env["ir.cron"]
+            .sudo()
+            .search_read(
+                [
+                    ("ir_actions_server_id.model_name", "=", "gamification.challenge"),
+                    ("code", "like", "._cron_update("),
+                ],
+                ["interval_type", "interval_number"],
```

Comment:
> and the cron record could have been replaced by another, right?

if it's the case we should just ignore it, we can't search based on a part of the code, that's super error prone

most of the time we take the XMLID, we do that for some template as well, etc (and it's ok)

> Handling the xmlid adds several requests

just for the first request no ? then it will be in cache and we won't do it again no ?